Graphite scaler docs (#524)
kedacore/keda-docs: content/docs/2.5/scalers/graphite.md

+++
title = "Graphite"
layout = "scaler"
availability = "v2.5+"
maintainer = "Community"
description = "Scale applications based on metrics in Graphite."
go_file = "graphite_scaler"
+++

### Trigger Specification

This specification describes the `graphite` trigger that scales based on metrics in Graphite.

```yaml
triggers:
- type: graphite
  metadata:
    # Required
    serverAddress: http://<graphite-host>:81

>     metricName: request-count # Note: metric sort name in query

Reviewer: ```suggestion
    metricName: request-count # Note: name to identify the Metric
```

    query: stats.counters.http.hello-world.request.count.count # Note: query must return a vector/scalar single element response
    threshold: '100'
    queryTime: '-10Minutes' # Note: Query time in from argv Seconds/Minutes/Hours
```
**Parameter list:**

- `serverAddress` - Address of Graphite
- `metricName` - Name to identify the Metric in the external.metrics.k8s.io API. If using more than one trigger it is required that all `metricName`(s) be unique
- `query` - Query to run
- `threshold` - Value to start scaling for

> - `queryTime` - Relative time range to execute query against.

Reviewer: Might be nice to add link to https://graphite-api.readthedocs.io/en/latest/api.html#from-until
